src/components/Calculator.js

```diff
@@ -1,40 +1,51 @@
-import React from 'react';
-import CalculatorButton from './CalculatorButton';
+import { useState, React } from 'react';
+import calculate from '../logic/calculate';
 
 function Calculator() {
+  const [objData, setObjData] = useState({
+    total: 0,
+    next: null,
+    operation: null,
+  });
+
+  const handleClick = (value) => {
+    const newObject = calculate(objData, value);
+    setObjData(newObject);
+  };
+
   return (
     <div className="CalculatorContainer">
       <div className="row" id="results">
-        0
+        {objData.next === null ? objData.total : objData.next}
       </div>
       <div className="row">
-        <CalculatorButton label="AC" />
-        <CalculatorButton label="+/-" />
-        <CalculatorButton label="%" />
-        <CalculatorButton label="÷" className="orange" />
+        <button type="button" onClick={() => handleClick('AC')}>AC</button>
+        <button type="button" onClick={() => handleClick('+/-')}>+/-</button>
+        <button type="button" onClick={() => handleClick('%')}>%</button>
+        <button type="button" className="orange" onClick={() => handleClick('÷')}>÷</button>
```

Comment:
- [ OPTIONAL ] For better user experience I would suggest to display the symbol that the user is clicking for example to show on calculator 5 - 3 or 5 + 3